src/components.d.ts

```diff
@@ -34,11 +34,11 @@ export namespace Components {
          */
         "checked": boolean;
         /**
-          * Disables the check box on the interface. If the attribute’s value is undefined, the value is set to false.
+          * Disables the checkbox on the interface. If the attribute’s value is undefined, the value is set to false.
          */
         "disabled": boolean;
         /**
-          * Label displayed on the interface, for the check box.
+          * Label displayed on the interface for the checkbox.
```

Comment:
Certain prepositions, such as for or on, have a tendency to gel (incorrectly) with the nouns immediately preceding the prepositions. Document users who are used to synthetic languages (inflections) might have a problem comprehending this kind of sentence construction. 
Could we retain the comma, to indicate that the prepositional phrase is superfluous information and to prevent the statement from meaning that the **interface** is **for the component**? 
Alternatively, we can say `Label for the component, displayed on the interface.` Like I mentioned, “for the component” is superfluous information and going by the principle of “important information first”, I suggest that we leave it at the end of the sentence.